system-test/spanner.ts

```diff
@@ -921,10 +918,10 @@ describe('Spanner', () => {
             return str.replace(/\n\s*/g, '').replace(/\s+/g, ' ');
           }
 
-          database.getSchema((err, statements) => {
+          DATABASE.getSchema((err, statements) => {
             assert.ifError(err);
             assert.strictEqual(
-              replaceNewLinesAndSpacing(statements![0]),
+              replaceNewLinesAndSpacing(statements![1]),
```

Comment:
Ah, this is tricky then. I don't think any test should depend on a previous execution to be kept in order. For example, if we add a test before this one that adds to the schema history, this would be bumped out of position. Other than creating a new database just for this test, will "statements.pop()" wrok?

Reply:
Adjusting it to check that the statement of interest does exist in the the `statements` array.